catalogData/redis32/ha/3x/k8s/sentinel-statefulset.json

```diff
@@ -0,0 +1,102 @@
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "StatefulSet",
+  "metadata": {
+    "name": "redis-sentinel"
+  },
+  "spec": {
+    "replicas": 3,
+    "serviceName": "redis-sentinel",
+    "template": {
+      "metadata": {
+        "annotations": {
+          "pod.beta.kubernetes.io/init-containers": "[{ \"name\": \"k8s-redis-ha-sentinel\", \"image\": \"tarosky/k8s-redis-ha:sentinel-3.0.1\", \"volumeMounts\": [{\"name\": \"opt\", \"mountPath\": \"/opt\"}] }]"
```

Comment:
As of k8s 1.6, we can specify init containers without this janky json-in-json or json-in-yaml syntax. Check out https://github.com/18F/kubernetes-broker/pull/53/files#diff-0641432dd9e1171e7af80325936f484fR53.